crowd.properties

```diff
@@ -1,25 +1,19 @@
 category=Integration
 description=Provide the ability to authenticate again Crowd
-homepageUrl=https://github.com/SonarCommunity/sonar-crowd
+homepageUrl=https://github.com/deepy/sonar-crowd
 devVersion=
-publicVersions=2.1,2.1.1
-archivedVersions=1.0,2.0
+publicVersions=2.1.2
+archivedVersions=1.0,2.0,2.1,2.1.1
 privateVersions=
 
 defaults.mavenGroupId=cm.xd.sonar-plugins
 defaults.mavenArtifactId=sonar-crowd-plugin
 
-2.1.1.description=Support SonarQube 7.3
-2.1.1.sqVersions=[7.3,LATEST]
-2.1.1.date=2018-08-23
-2.1.1.changelogUrl=https://github.com/deepy/sonar-crowd/releases/tag/2.1.1
-2.1.1.downloadUrl=https://github.com/deepy/sonar-crowd/releases/download/2.1.1/sonar-crowd-plugin-2.1.1.jar
-
-2.1.description=Support SonarQube 6.7-7.2
-2.1.sqVersions=[6.7,7.2]
-2.1.date=2018-08-21
-2.1.changelogUrl=https://github.com/deepy/sonar-crowd/releases/tag/2.1
-2.1.downloadUrl=https://github.com/deepy/sonar-crowd/releases/download/2.1/sonar-crowd-plugin-2.1.jar
+2.1.2.description=Support SonarQube 6.7-7.7
+2.1.2.sqVersions=[6.7,7.3,LATEST]
```

Comment:
@deepy the `[...]` syntax describes a range, and ranges only accept 2 vaues: start and end. :-D.

Did you mean: [6.7,6.7.*],[7.3,LATEST]
i.e. all point releases of 6.7 and everything after 7.3
?

Reply:
Oops, will change it to 6.7 -> LATEST. I've tested it with 6.7, 7.3, and 7.7